lessons/Early-Math/associative-property/langs/en-us/index.adoc

```diff
@@ -0,0 +1,123 @@
+
+=== Launch
+
+When asked to multiply @math{2 \times 17 \times 5}, the student who notices that @math{2 \times 5 = 10} will arrive at a result much more efficiently than a student who does not. This is the power of the @vocab{Associative Property}!
+
+In the short term, familiarity with the Associative Property enables students to make mental computations more efficiently. In the long term, comfort with this law of arithmetic (and others!) will provide a strong foundation for more complex algebraic reasoning.
+
+A note: We view the Associative Property as the @vocab{Commutative Property} *2.0*! With the Associative Property, students can not only change the order of the values in a multiplication or addition expression... they can change the _groupings_. _Finding the best groupings, students will discover, sometimes requires changing the order as well._
+
+[.lesson-instruction]
+- Turn to @printable-exercise{assoc-prop-launch.adoc}.
+- Look at the expression at the top of the page: @math{88 + 71 + 637 + 129 + 12}
+- How many different ways can you re-group and re-order the expression? Draw as many equivalent circles of evaluation as you can in the boxes below, then answer the questions at the bottom of the page.
+- As you complete the Circles of Evaluation, strive for groupings that will make the computation simpler.
+
+Allow students to work, encouraging them to try not only different groupings but different _structures_ as well - all of which will produce the same outcome! If possible, invite students to draw their Circles of Evaluation on the board, so that the class can see all possible Circles. After students have drawn their Circles of Evaluation and had some time to consider the reflection questions, follow-up with a class discussion.
+
+[.lesson-instruction]
+- Which Circle of Evaluation seemed like it would be the most difficult to solve in your head? Why?
+- Which Circle of Evaluation seemed like it would be the easiest to solve in your head? Why?
+- Did you think that any Circles of Evaluation were _equally_ easy or difficult to evaluate? What was going on in those Circles of Evaluation?
+** _Students who applied the Commutative Property - merely changing the order of two numbers in the same group - would encounter Circles of Evaluation that were equally easy / challenging to evaluate._
+
+
+=== Investigate
+
+Explain to students that they have just made use of the @vocab{Associative Property} of Addition!
+
+Let's continue to explore the impact of changing the _groupings_ of an @vocab{arithmetic expression} to understand when we can and cannot apply the Associative Property.
+
+[.lesson-instruction]
+- Turn to @printable-exercise{discover-assoc.adoc}. Look at the first problem.
+- What do you Notice about the two Circles of Evaluation? How about the two expressions that accompany them? What do you Wonder?
```

Comment:
If you make the change I requested on the workbook page, be sure to change the wording of this question from two to three.